### DISCUSSION TITLE
New supported plugin

### DISCUSSION THREAD

**Discussion by Grazen0** [games647/FastLogin#460] [Q&A] [answered]
Hey there! I'm the dev of the [UserLogin](https://www.spigotmc.org/resources/userlogin.80669/) plugin, and just added support for FastLogin to it. Wanted to ask if my plugin could be added to the "Supported plugins" list.

It's not really a problem if it's not possible, but I wanted to consider the possibility :D

**Comment by TuxCoding** [comment, marked as answer]
Added. I missed this post sorry. BTW: discussions are more like a forum. So for requests, please use the tickets. It's a little more straight forward.

**Reply by Grazen0** [reply]
Understood, thanks for the response.
